src/pages/Register.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-
+import RegisterForm from '../components/Auth/RegisterForm';
```

Comment:
Let's make all file and folder names lower case and separate words with a dash

Reply:
🔥 I'll update worksheets to reflect that